docs/openshift/kctlr-openshift-routes.rst

```diff
@@ -36,41 +36,122 @@ When you use the |kctlr| as a `Router`_, you can
    =======  ===================================================================
    1.       :ref:`set up kctlr routes`
 
+            - :ref:`route existing virtual`
+
    2.       :ref:`create os route`
 
-   3.       :ref:`route-TLS` (OPTIONAL)
+   3.       :ref:`deploy route resource`
+
+   4.       :ref:`verify BIG-IP route objects`
 
-   4.       :ref:`add health monitor to route` (OPTIONAL)
+   5.       :ref:`attach bigip objects routes`
 
-   5.       :ref:`deploy route resource`
+            - :ref:`add health monitor to route`
+            - :ref:`route-TLS`
+            - :ref:`delete vs route`
 
-   6.       :ref:`verify BIG-IP route objects`
    =======  ===================================================================
 
 .. _set up kctlr routes:
 
-Set up the BIG-IP Controller to manage Routes
----------------------------------------------
+Deploy the BIG-IP Controller
+----------------------------
 
-If you haven't already done so, add the |kctlr| `Route configuration parameters`_ to the |kctlr| Deployment:
+.. include:: /_static/reuse/kctlr-openshift-deployment-note.rst
 
-.. literalinclude:: /openshift/config_examples/f5-kctlr-openshift-routes.yaml
-   :linenos:
-   :emphasize-lines: 47-56
+#. Create a Kubernetes Deployment using valid YAML or JSON. Define the |kctlr| `Route configuration parameters`_ as appropriate to suit your needs.
+
+   .. literalinclude:: /openshift/config_examples/f5-k8s-bigip-ctlr_openshift_routes.yaml
+      :linenos:
+
+   .. warning::
+
+      Use caution when setting the :code:`--route-vserver-addr` and :ref:`specifying a BIG-IP SNAT pool <kctlr-openshift snat deploy>`.
+
+      If you choose to set both options, make sure the IP address defined for the virtual server falls within the range of the selected SNAT pool.
+
+#. :ref:`upload openshift deployment`.
+
+.. _route existing virtual:
+
+Attach a Route to an Existing Virtual Server
+````````````````````````````````````````````
+
+.. include:: /_static/reuse/k8s-version-added-1_5.rst
+
+Take the steps below to attach a Route to an existing BIG-IP virtual server.
```

Comment:
Ignoring my lack of formatting and phrasing, something like the following might work better (also, we removed the need to specify the virtual address, apparently it wasn't a problem):
```
If you need to customize a virtual server associate with Routes in a way that is not natively supported by the controller, you can perform the following steps:
1. Pre-create the virtual server in the partition you want to manage before starting the controller.
2. Add any customization necessary for the virtual that are compatible with the controller (see warning below).
3. Use the TMSH shell to set the 'cccl-whitelist' metadata field.  This field will indicate to the controller that it should merge its configuration into the existing virtual instead of replacing it.  
example:  modify ltm virtual [myVirtual] metadata add { cccl-whitelist { value 1 }}
4. Start the controller and specify the name of the virtual using either `route-http-vserver` for an HTTP virtual server or :code:`route-https-vserver` for an HTTPS virtual server. (show example)
```